File: Solutions/Summation_of_primes.cpp
#include <iostream>
using namespace std;
int main()
{
    int n,sum=0,i,j,c=0;
    cin>>n;
    for(i=2;i<=n;i++)
    {
    	c=0;
    	for(j=2;j<i;j++)
    	{
    		if(i%j==0)
    			break;
    		else
    			c++;
		}
		if(c==(i-2))
		{
			sum=sum+i;
		}
	}
    cout<<sum;
}

File: Solutions/Summation_of_primes.rb
def get_primes(num)
  start = 2
  primes = (start..num).to_a 

  (start..num).each do |i|
    (start..i).each do |ii|
      if (i % ii == 0) && i != ii
        primes.delete(i)
        break
      end
    end
  end
  primes.sum
end

print 'Please Enter a number? '
num = gets.chomp
puts get_primes(num.to_i)

